### ISSUE TITLE
Change default branch from master to main

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris-react#3758]

**Comment by kaelig** [comment]
:+1: This will be unblocked by this project: https://vault.shopify.io/projects/14993

**Comment by alex-page** [comment]
I would love to see solutions instead of blocking labels and comments. That project seems to be blocked and not progressing. I have reached out to @flavorjones.

> Hey Mike, just wanted to let you know that GitHub now has made main the default for new repositories. Should we move forwards with Support Renaming the Git Default Branch Name.

What is stopping us from renaming the branch on this open source repo? I would assume we aren't blocked by Shopify tooling as all of our users do not have access to that tooling.

**Comment by kyledurand** [comment]
This was actually something I suggested while chatting with Alex so I'll chime in. 

I disagree that we're blocked by that project, especially if it's not progressing as quickly as we'd like. With all that's been on our minds this year I think this change is important and if we can get it done in our projects maybe other teams will follow suit.

**Comment by kaelig** [comment]
Hopefully, this blocker should only delay this change by a few weeks, as GitHub will roll out a few changes in January 2021 that'll make the transition easier (which is when I'd expect that project — which is "on hold", to restart).

Another potential blocking factor: last I checked, some of our internal tooling such as the translation platform doesn't work with branches named anything other than master.

**Comment by alex-page** [comment]
I don't mind if it's a slightly harder transition as we have to update some base branches. I think it's probably worth it.

@kyle and I can take a look at translations.

**Comment by kaelig** [comment]
As one of the people who started this internal initiative, my recommendation is to wait a bit and make this change as part of a concerted effort, in tandem with the rest of the company. This decision has been approved by various senior engineering leaders — perhaps Mike can provide some more nuance on that.

What could be useful is to offer our help as a pilot repository, knowing that Polaris isn't a critical service and it'd be okay if some of the tooling around it broke, as it probably wouldn't impact our ability to ship new versions of the library.

It might be a good idea to look at migrating some simpler repositories first, such as polaris-icons and polaris-tokens, as they depend on very little tooling and would be a good way to predict what sort of issues we'd run into with polaris-react.

**Comment by flavorjones** [comment]
I connected with Alex on Slack today. To summarize: I'd prefer if teams waited until Github published a plan and updates their tooling; and until Shopify can then update any of our internal tools that need to accommodate.

That said, individual teams are obviously empowered to make this change to their repos, if they understand the limitations and risks and don't have any expectations from Dev Accel for tooling updates yet.

**Comment by movermeyer** [comment]
I see that this repo's default branch just changed to `main`....

🤞 things don't break.
